src/snowflake/snowpark/__init__.py

```diff
@@ -34,6 +34,7 @@
     "WhenNotMatchedClause",
     "QueryRecord",
     "QueryHistory",
+    "AsyncJob",
```

Comment:
You will need to add this class here to make doc working https://github.com/snowflakedb/snowpark-python/blob/main/docs/source/_templates/module.rst